Testing/Unit/Rego/AAD/AADConfig_05_test.rego

```diff
@@ -0,0 +1,33 @@
+package aad
+import future.keywords
+import data.report.utils.notCheckedDetails
+
+#
+# Policy 1
+#--
+test_NotImplemented_Correct_V1 if {
+    PolicyId := "MS.AAD.5.1v1"
+
+    Output := tests with input as { }
+
+    RuleOutput := [Result | Result = Output[_]; Result.PolicyId == PolicyId]
+
+    count(RuleOutput) == 1
+    not RuleOutput[0].RequirementMet
+    RuleOutput[0].ReportDetails == notCheckedDetails(PolicyId)
+}
+
+#
+# Policy 2
```

Comment:
There is not a good mapping on some of the new markdown policy id(s) to the old control/requirement combination.  I removed the policy 2 comment since it is associated with old style and no longer has meaning in new context.